pkg/types/types.go

```diff
@@ -57,6 +57,8 @@ type Options interface {
 	// returns the path to the directory where artifacts should be written
 	// (including metadata files like junit_runner.xml)
 	ArtifactsDir() string
+	// UUID returns a unique identifier for a kubetest2 run.
+	UUID() string
```

Comment:
Would `RunID` be a more descriptive identifier?

Reply:
done